packages/expo-modules-core/android/src/main/cpp/JavaScriptValue.cpp

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2021-present 650 Industries, Inc. (aka Expo)
+
+#include "JavaScriptValue.h"
+
+#include "JavaScriptRuntime.h"
+#include "JavaScriptObject.h"
+
+namespace expo {
+void JavaScriptValue::registerNatives() {
+  registerHybrid({
+                   makeNativeMethod("kind", JavaScriptValue::jniKind),
```

Comment:
Indenting like this would probably have more sense:
```suggestion
    makeNativeMethod("kind", JavaScriptValue::jniKind),
```